map-storage/src/index.ts

```diff
@@ -22,15 +27,23 @@ server.bindAsync(`0.0.0.0:50053`, grpc.ServerCredentials.createInsecure(), (err,
 const app = express();
 app.use(cors());
 
-app.get("/", (req, res) => {
-    res.send("Hello World!");
-});
+passport.use(passportStrategy);
+app.use(passport.initialize());
+//app.use(passport.session());
 
 app.get("*.json", async (req, res) => {
-    res.send(await mapsManager.getMap(req.url));
+    const path = req.url;
+    const domain = req.hostname;
+    if (path.includes("..") || domain.includes("..")) {
+        res.status(400).send("Invalid request");
+        return;
+    }
+    res.send(await mapsManager.getMap(path, domain));
```

Comment:
If there are error in `mapsManager`, what's happening? No data returned?

Might we manage error et return status with body?

Reply:
Oooooh good point!
I put an exception just aboe to see how Express behaves when not catching an error.....

The result? NodeJS crashes :O